{"uuid": "7bb7c6aa-38d1-4ffa-819a-b0738409ba54", "children": ["12888251-9729-4c50-9cc0-b101a56ace20"], "befores": [{"name": "country", "status": "broken", "statusDetails": {"message": "AttributeError: module 'selenium.webdriver.chrome.webdriver' has no attribute 'Firefox'\n", "trace": "  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\Tests\\conftest.py\", line 109, in country\n    country = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n              ^^^^^^^^^^^^^^^^^\n"}, "start": 1730823657118, "stop": 1730823657118}], "start": 1730823657118, "stop": 1730823657306}